content/blogs/release-0-22.md

```diff
@@ -0,0 +1,286 @@
+
+The new read-only mode for external secrets managers allows you to reference secrets that remain managed in your external system. The integration currently supports HashiCorp Vault with path prefix capabilities, enabling you to precisely control which secrets are accessible to your workflows.
+
+TO CHECK: WHAT ABOUT GCP, AWS, Azure?
+
+The UI clearly distinguishes externally managed secrets with a lock icon, providing visual confirmation of their read-only status. These secrets cannot be edited, created, or deleted through Kestra, ensuring your security policies remain enforced at the source.
+
+MORE DETAILS TBD
+
+![](SCREENSHOT 2)
+
+
+
+### `afterExecution` property
+
+This release introduce a new flow property: `afterExecution`. This allows to run any set of tasks **after** the execution of the flow. It differs from the `finally` property that run tasks at the end of the flow while the execution is still in `RUNNING` state.
+
+You might use `afterExecution` to send notifications or update documentation after a flow completes, regardless of whether it succeeded or failed. Unlike `finally` which runs while the flow is still active, afterExecution ensures these tasks only begin after the entire execution finishes, providing cleaner separation between your core workflow and post-completion tasks.
+
+::collapse{title="Example of `afterExecution` vs `finally`"}
+```yaml
+id: state_demo
+namespace: company.team
+
+tasks:
+  - id: run
+    type: io.kestra.plugin.core.log.Log
+    message: Execution {{ execution.state }} # Execution RUNNING
+  
+  - id: fail
+    type: io.kestra.plugin.core.execution.Fail
+
+finally:
+  - id: log 
+    type: io.kestra.plugin.core.log.Log
+    message: Execution {{ execution.state }} # Execution RUNNING
+
+afterExecution:
+  - id: log_after
+    type: io.kestra.plugin.core.log.Log
+    message: Execution {{ execution.state }} # Execution FAILED
+```
+::
+
+
+
+### Sharing of Namespace Files and KV Store across namespaces
+
+- https://github.com/kestra-io/kestra/issues/5467
+
+
+
+NamespaceFiles
+
+```yaml
+id: hyena_375528
+namespace: dev.ben
+
+tasks:
+  - id: hello
+    type: io.kestra.plugin.core.log.Log
+    message: Hello World! 🚀
+
+  - id: ns
+    type: io.kestra.plugin.scripts.python.Commands
+    namespaceFiles:
+      enabled: true
+      namespaces:
+        - "company"
+        - "dev.ben"
+    commands:
+      - python test.py
+```
+
+## User Interface & Experience Improvements
+
+As with each release, there are more UI and UX enhancements:
+
+- Improved Editor contrast in the light mode
+- Export topology view to PNG and JPG
+- Improvements to flow filters in the UI (Filter flows by text, filter by multiple labels)
+- As part of our continuous improvements to the No Code experience, we're releasing a Beta version of a multi-tab system in the editor. To enable this feature, navigate to Settings > Multi-tab Editor.
```

Comment:
I was unable to find this in `develop`. Just want to make sure this is still in progress and not being missed

Reply:
- I tested export topology to PNG and JPG last week, should be ok
- there is an issue with multi label, I've created a QA issue https://github.com/kestra-io/kestra/issues/8016
- No Code multi-tab is not merged yet, but will ultimately available this week (will remove if not the case

Reply:
This is visible now